recipes/xlnt/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class XlntConan(ConanFile):
+    name = "xlnt"
+    description = "Cross-platform user-friendly xlsx library for C++11+"
+    license = "MIT"
+    topics = ("excel", "xlsx", "spreadsheet", "reader", "writer")
+    homepage = "https://github.com/tfussell/xlnt"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("libstudxml/1.1.0-b.10+1")
+        self.requires("miniz/3.0.1")
+        self.requires("utfcpp/3.2.2")
+
+    def validate(self):
+        if self.info.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, 11)
+        libstudxml_version = Version(self.dependencies["libstudxml"].ref.version)
+        libstudxml_major_minor = f"{libstudxml_version.major}.{libstudxml_version.minor}"
+        if Version(libstudxml_major_minor) < "1.1":
```

Comment:
I was uned the impression the new version helper should ignore the prerelease and build information 🤔 
Did you try without this or just went to try and trusted 1.x impl?

Curious if there's a bug :wink

Reply:
Thank you for checking!